modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/odata/entity/v1_0/KeywordEntityModel.java

```diff
@@ -33,7 +33,8 @@ public KeywordEntityModel() {
 				locale -> Field.getSortableFieldName(Field.MODIFIED_DATE),
 				locale -> Field.MODIFIED_DATE),
 			new IdEntityField(
-				"groupIds", locale -> "groupIds", String::valueOf),
+				"assetTagGroupRelsGroupIds",
```

Comment:
The first param should remain `groupIds`. It needs to match the entity's field name defined in the schema. The second param maps the field to the indexed field.